pkgs/data/fonts/liberation-fonts/default.nix

```diff
@@ -20,7 +20,10 @@ let
         installPhase = ''
           find . -name '*.ttf' -exec install -m444 -Dt $out/share/fonts/truetype {} \;
 
-          install -m444 -Dt $out/share/doc/${pname}-${version} ${lib.concatStringsSep " " docsToInstall}
+          for i in ${toString docsToInstall}; do
```

Comment:
Replace with?

```
# not all docs exist in all versions
install -m444 -Dt $out/share/doc/${pname}-${version} ${lib.concatStringsSep " " docsToInstall} || true
```

Reply:
I'm not sure how that will work with installing multiple files, I think this way it's at least clear what we're trying to do.

Reply:
I agree that getting rid of «does `install` continue after first failure» even if it's pretty likely a «yes» is a reasonable trade-off